content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -265,9 +266,9 @@ executes the files in the following order:
 
 ```html
```

Comment:
I have some updates coming to this section below where we're displaying both e2e & ct info with a tabbed UI. Not biggie, just a heads up.

https://github.com/cypress-io/cypress-documentation/pull/4280/commits/c6fce89769e18351507d454856891504e805db1d

Reply:
If possible, why don't we split this into the text changes to spec extensions you have and then you can cherry-pick this commit that I have in flight. https://github.com/cypress-io/cypress-documentation/commit/c6fce89769e18351507d454856891504e805db1d 

I have more work on this PR and that will just avoid conflicts down the line.